plutus-tx-plugin/test/Blueprint/Acme.golden.json

```diff
@@ -67,13 +67,17 @@
     "Datum": {
       "oneOf": [
         {
-          "$comment": "DatumLeft",
+          "title": "Datum",
+          "description": "DatumLeft",
```

Comment:
seems wrong

Reply:
I would expect something like
```
"title": "DatumLeft",
"description": "Description about the constructor, basically it's haddock"
```

Reply:
But I can't really engage with this because you haven't written down your reasoning for assigning annotations in this way. Write it down!